Header header

# z
float32 posZ 
float32 rawPosZ
float32 velZ
float32 rawVelZ
float32 crrPosZ1
float32 crrVelZ1
float32 crrPosZ2
float32 crrVelZ2
float32 crrBias
float32 emptyZ1
float32 emptyZ2




